### ISSUE TITLE
Use `prop-types` instead of React.PropTypes for Accordion component

### ISSUE THREAD

**Issue by nkahoang** [digiaonline/react-foundation#42]
### What did you do?
Using react-16beta with react-foundation failed because react-foundation's Accordion components was using `React.PropTypes`, which is deprecated in React v15.x and removed in v16; instead of the recommended way of using the dedicated package: `prop-types`

### What did you expect to happen?
Accordion to use `prop-types` package instead of React.PropTypes

### What version of this module are you using?
0.8.0 - 0.8.2 since Accordion is introduced

Please refer to https://github.com/facebook/react/issues/10294#packaging

Thank you

**Comment by hugovk** [comment]
This should have been fixed in https://github.com/digiaonline/react-foundation/pull/34 and released in version 0.8 https://github.com/digiaonline/react-foundation/pull/39.

https://github.com/digiaonline/react-foundation/blob/master/src/components/accordion.js#L2 has:

```js
import PropTypes from 'prop-types';
```

@nkahoang Please can you double check what version you're using?

**Comment by hugovk** [comment]
I just did an `npm i react-foundation` and the installed node_modules/react-foundation/src/components/accordion.js has:
```js
import PropTypes from 'prop-types';
```

Please update the version you're using.
